addons/payment_stripe/controllers/main.py

```diff
@@ -171,3 +169,40 @@ def _verify_webhook_signature(self, webhook_secret):
             return False
 
         return True
+
+    # -------------------------------------------
+    # Stripe Connect Onboarding
+    # -------------------------------------------
+    @http.route('/payment/stripe/onboarding/return/<int:acquirer_id>', type='http', auth='public', csrf=False)
+    def stripe_onboarding_return(self, acquirer_id, **kwargs):
+        """Stripe return URL
+
+            This route is used by Stripe to return after or during the onboarding.
+            Once called, the controller tests the account status of the Onboarded Stripe account
+            and redirect to it.
+
+            :returns: Redirection to Stripe Acquirer Form
+        """
+        onboarding = kwargs.get('onboarding', 'payment')
```

Comment:
Why not explicitly put this kwarg in the method signature?

Reply:
I hesitated, I'll add it.